services/app-api/resolvers/configureResolvers.ts

```diff
@@ -22,6 +24,7 @@ export function configureResolvers(store: Store, emailer: Emailer): Resolvers {
             fetchCurrentUser: fetchCurrentUserResolver(),
             fetchDraftSubmission: fetchDraftSubmissionResolver(store),
             fetchStateSubmission: fetchStateSubmissionResolver(store),
+            fetchSubmission2: fetchSubmission2Resolver(store),
```

Comment:
just checking my understanding - is the goal is that `fetchSubmission2`  (or whatever we name it) would replace `fetchDatchSubmission` and `fetchStateSubmission` both?